### DISCUSSION TITLE
open second window inside of main one

### DISCUSSION THREAD

**Discussion by dongfanglong2012** [r0x0r/pywebview#821] [Q&A] [answered]
I have a project I'm making that needs to load another site that is logged into with a smart card into a small window inside my main one. It will load that site just fine but I cant embed it in a iframe as its security blocks it. So is there a way to open another pywebview and embed that into the main window? That would get around the iframe block and i wouldn't be bypassing any security setups. I found a way to force the site to load in an iframe anyway but that breaks the login system. 

[Here ](https://imgur.com/a/mWAtPd5) is a screenshot of what the general look is. Hope that better explains what I'm trying to do.

**Comment by r0x0r** [comment, marked as answer]
No, this is not possible, nor likely will be ever implemented.
You may try looking to disabling web security. This is possible at least with CEF, not sure about other renderers.
